api/python/provisioner/srv/salt/provisioner/files/post_setup.sh

```diff
@@ -87,5 +87,5 @@ test -e ${PWD}/api/python/setup.py && \
   install_dir=${PWD} || \
   install_dir=/opt/seagate/cortx/provisioner
 
-chown -R :"$prvsnr_group" "${install_dir}/cli"
-chmod -R 750 "${install_dir}/cli"
+#chown -R :"$prvsnr_group" "${install_dir}/cli"
```

Comment:
SUGGESTION: Lines can be removed if it's not going to be required again in future.
Rest looks ok.

Reply:
removed lines